### PR TITLE
refactor with ruby_rbenv-2.0.7 DEVOPS-1648

### PR DESCRIPTION
**Pull request by drew-holt**

### CODE REVIEW

**Comment by drew-holt** [comment]
[test]

**Comment by huynt1979** [comment]
@drew-holt You'll need to test this in Dev and QA to confirm it doesn't break anything.

File: attributes/system_ruby.rb [hunk 1]
```diff
@@ -23,19 +16,12 @@
...
 }
-default['rbenv']['upgrade'] = 'sync'
-
-# user rubies
-default['rbenv']['user_rubies'] = [
-  '2.3.5'
-]
-default['rbenv']['user_plugins'] = []
-default['rbenv']['user_gems'] = {}
+default['rbenv']['root_path'] = '/usr/local/rbenv'
```
Comment (huynt1979) [inline attributes/system_ruby.rb]: Unless a package is installed thru standard mechanism such as apt or yum, we dedicate /opt for pretty much anything that follows the pattern **download-extract-run**

Comment (drew-holt) [reply attributes/system_ruby.rb]: rbenv installs into /usr/local/rbenv by default, currently testing in dev

**Comment by drew-holt** [comment]
[testing]

**Comment by huynt1979** [comment]
Is it running in both Dev & QA?

**Pull request merged**